### ISSUE TITLE
Brackets in LaTeX Titles

### ISSUE THREAD

**Issue by Jahrme** [rbeezer/mathbook#363]
LaTeX titles (specifically short titles) need to be handled more carefully. Currently, if there are brackets in the `title` of a subdivision, the brackets wreak havoc on the title. The simplest solution is to wrap the short title in braces to hide the brackets.

Currently:
```
\subsection[A [short] title]{A [short] title}
```

Better:
```
\subsection[{A [short] title}]{A [short] title}
```

Minimal example (saved as `txt` due to GitHub rejecting `xml` and `mbx`):
[brackets-in-titles.txt](https://github.com/rbeezer/mathbook/files/404517/brackets-in-titles.txt)

(HTML does not appear to have any issues with brackets in titles.)

**Comment by Jahrme** [comment]
Fix at proposed PR #377

**Comment by Jahrme** [comment]
Solution (PR #377) accepted, closing issue.
